test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -1234,22 +1235,14 @@ class test_ucp_sockaddr_wireup_fail : public test_ucp_sockaddr_wireup {
             UCS_TEST_SKIP_R("cannot connect to server");
         }
 
-        if (fail_wireup_type == FAIL_WIREUP_MSG_SEND) {
-            emulate_failure(e, fail_wireup_type);
-        }
-
         ucs_time_t deadline = ucs::get_deadline();
         while (!test_all_ep_flags(e, wait_ep_flags) &&
-               (sender().get_err_num() == 0) && (ucs_get_time() < deadline)) {
+               (sender().get_err_num() == 0)) {
```

Comment:
maybe check m_err_count like in line 1213?

Reply:
done